lib/twitter_cldr/js/tasks/tasks.rb

```diff
@@ -51,28 +51,19 @@ def build(options = {})
           build_duration = time_operation do
             options[:files].each_pair do |file_pattern, minify|
               compiler.compile_each(:minify => minify) do |bundle, locale|
-                out_file = File.join(output_dir, file_pattern % locale)
-                FileUtils.mkdir_p(File.dirname(out_file))
-                File.open(out_file, "w+") do |f|
-                  f.write(bundle.source)
-                end
+                out_file_path = [output_dir, file_pattern % locale]
+                write_bundle(out_file_path, bundle.source)
```

Comment:
It's up to you, but I don't think this `write_bundle` method adds much value. I'd just do inline 

```
write_file(File.join[output_dir, file_patter % locale], bundle.source)
```

here and call `write_file` instead. Same below. 

Besides, looks like `File.join` is quite intelligent and can eat pretty much anything:

```
> File.join('tmp/qw.js')
 => "tmp/qw.js"
> File.join(['tmp/qw.js'])
 => "tmp/qw.js"
> File.join(['base', 'tmp/qw.js'])
 => "base/tmp/qw.js"
> File.join('base', 'tmp/qw.js')
 => "base/tmp/qw.js"
```
so you probably can just pass this array straight into `write_file` and there blindly apply `File.join` to whatever came in in `file_name`. I'd still prefer an explicit `File.join` though, just to be safe.